test/service/storage/elasticsearch.test.js

```diff
@@ -3536,6 +3536,36 @@ describe('Test: ElasticSearch service', () => {
       elasticsearch._mExecute = sinon.stub().resolves(mExecuteResult);
     });
 
+    it('should call _mExecute with formated documents and source flag', async () => {
```

Comment:
My idea is to make like in redis.test.js ([line 68](https://github.com/kuzzleio/kuzzle/blob/6cbf4ad03cb5a2923863771451d1e360043e8776/test/service/cache/redis.test.js#L68)).

The goal is to avoid nested then calls like:

```js
const result = await elasticsearch.mCreateOrReplace(index, collection, documents, { source: false });
```